packages/edit-post/src/components/meta-boxes/index.js

```diff
@@ -30,14 +30,16 @@ export default function MetaBoxes( { location } ) {
 		[ location ]
 	);
 
+	const hasMetaBoxes = metaBoxes?.length > 0;
```

Comment:
I find these checks a bit weird(possible `undefined > 0`) and I think that `!! metaBoxes?.length` is clearer, but that's a super nit and a personal preference.